content/zh-cn/docs/tasks/configmap-secret/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: "管理 Secrets"
-weight: 28
+weight: 60
```

Comment:
The frontmatter in english can be copied and commented here.

Reply:
updated